server/activityItem/fetch.ts

```diff
@@ -198,7 +198,9 @@ const getActivityItemAssociationIds = (
 			thread.add(item.payload.threadId);
```

Comment:
These few changes relating to `threadcomment.commenterId` are not strictly related to the fix — they're downstream from a change to our TS `ThreadComment` type where we now assert that both `userId` (for a real PubPub user) or `commenterId` (for a guest commenter) can be nullable.